tools/configure_for_migration_analytics/tasks/install-vddk.yml

```diff
@@ -0,0 +1,39 @@
+---
+- name: CFME | Install VDDK | Create Temp Directory
```

Comment:
This feels ore like a general tool that

- The entire project would benefit from, not just migration analytics
- should live in the manageiq-providers-vmware plugin

cc @agrare

Reply:
It also feels like this should just be a dedicated ansible role

Reply:
I agree that a dedicated role would be ideal, ideally with UI support (or a shipped service and service dialog) so that users can input the required extra_vars. Should I reopen [this](https://bugzilla.redhat.com/show_bug.cgi?id=1729521) as a new issue in manageiq-providers-vmware?